.github/autobuild/mac.sh

```diff
@@ -146,6 +180,26 @@ pass_artifact_to_job() {
     echo "Moving build artifact to deploy/${artifact}"
     mv ./deploy/Jamulus-*installer-mac.dmg "./deploy/${artifact}"
     echo "artifact_1=${artifact}" >> "$GITHUB_OUTPUT"
+
+    artifact2="jamulus_${JAMULUS_BUILD_VERSION}_mac${ARTIFACT_SUFFIX:-}.pkg"
+    file=(./deploy/Jamulus_*.pkg)
```

Comment:
Maybe this needs quotes?
@softins

Reply:
The shell should deal with it -- standard bash file expansion into an array.